### ISSUE TITLE
[Feature Request] Option to delete devices from "Network" tab

### ISSUE THREAD

**Issue by ferferga** [pi-hole/AdminLTE#1044]
Before starting, thank you very much for bringing this piece of software for everybody!

I would like to delete some devices that appear under the "Network" tab in my Pi-hole dashboard, and, although I know how to do it altering the database, probably many other users are looking forward this feature

Would be great to have it in the web UI!

**Comment by pralor** [comment]
This issue has been mentioned on **Pi-hole Userspace**. There might be relevant details there:

https://discourse.pi-hole.net/t/remove-orphan-clients-from-pi-hole-update-dns-names/32451/9

**Comment by ferferga** [comment]
This is already on pihole, so I'm closing this

**Comment by boomam** [comment]
I am not sure that this feature works correctly.

Is anyone able to explain in detail what is expected to happen? 
As simple as pressing the 'flush network table' and the client list in the logs elsewhere in the system are clean of old client names? 
As if so, that doesn't appear to work for me...

**Comment by ferferga** [comment]
@boomam No, that button just clears the network table

**Comment by boomam** [comment]
Ok, then this feature should be reopened as it has not solved the OP's request.
The OP asked _Option to delete devices from "Network tab_, and as you confirmed the button does not do that, it clears the network table only. 
I can confirm that the same button does not do the function that the OP is asking as a by-product either.

As such, this feature should be reopened and dealt with accordingly.

**Comment by rdwebdesign** [comment]
We receive a high number of requests, but as a free open-source project run by volunteers we have very limited resources.
Please, we ask that you open your Feature Requests at our **[Discourse Forum](https://discourse.pi-hole.net/c/feature-requests/8)**.

Thank you for your understanding.

---

**Pi-hole links:**
- Discourse Forum: https://discourse.pi-hole.net/
- Documentation: https://docs.pi-hole.net/
- Feature Requests: https://discourse.pi-hole.net/c/feature-requests/8

**Comment by yubiuser** [comment]
> The OP asked Option to delete devices from "Network tab, and as you confirmed the button does not do that, it clears the network table only.

I'm not sure what you mean. I think there is no other "Network tab" than the the Network table in "Tools/Network"

While "Flush network table" flushes all of them, this PR https://github.com/pi-hole/AdminLTE/pull/1924 has implemented the function to delete each entry separately. Please note that they will be re-added quite quickly if the device is still on your network.

**Comment by boomam** [comment]
> > The OP asked Option to delete devices from "Network tab, and as you confirmed the button does not do that, it clears the network table only.
> 
> I'm not sure what you mean. I think there is no other "Network tab" than the the Network table in "Tools/Network"
> 
> While "Flush network table" flushes all of them, this PR #1924 has implemented the function to delete each entry separately. Please note that they will be re-added quite quickly if the device is still on your network.

I'd be surprised if the OP was bothered about the location, but more so that the function exists.

For myself, your linked method does not work whatsoever.

**Comment by yubiuser** [comment]
> For myself, your linked method does not work whatsoever.

Then please open an issue, preferable on https://discourse.pi-hole.net/

**Comment by ferferga** [comment]
@boomam The button did exactly what I wanted: clear the network table completely.

Maybe I worded my original message badly.

Of course, removing individual devices is another (completely separate) fracture request.

**Comment by rdwebdesign** [comment]
The OP closed this issue with **"This is already on pihole, so I'm closing this"**.
The OP answered to you, explaining: **"No, that button just clears the network table"**.

If you have a new issue, please open an issue on Discourse.

**Comment by boomam** [comment]
> @boomam The button did exactly what I wanted: clear the network table completely.
> 
> Maybe I worded my original message badly.
> 
> Of course, removing individual devices is another (completely separate) fracture request.

Great!
And yes, good point.
